pkg/cacheutil/memcached_client.go

```diff
@@ -340,11 +345,16 @@ func (c *memcachedClient) SetAsync(_ context.Context, key string, value []byte,
 		start := time.Now()
 		c.operations.WithLabelValues(opSet).Inc()
 
+		span, _ := tracing.StartSpan(ctx, "memcached_client_set")
```

Comment:
Note that by its async nature, this span will appear later in the trace, after its parent has already finished. AFAIK most tracing tools should handle this case ok.